tools/slang-embed/slang-embed.cpp

```diff
@@ -92,13 +94,99 @@ struct App
         // could lead to a difference in the embedded bytes based on
         // the line ending convention of the host platform)
         //
-        FILE* inputFile = fopen(inputPath, "r");
+        Slang::StreamReader streamReader(inputPath);
+        FILE* inputFile = fopen(inputPath.getBuffer(), "r");
```

Comment:
Is there a reason the `inputFile` variable is still being used?

Reply:
Good catch.